doc/api/http.md

```diff
@@ -622,7 +622,7 @@ Once a socket is assigned to this request and is connected
 added: v0.5.9
 -->
 
-* `timeout` {number} Milliseconds before a request is considered to be timed out.
+* `timeout` {number} Milliseconds before a request times out.
```

Comment:
Unrelated but I think this description is a bit misleading. This timeout expires only after `x` millisecond of inactivity on the socket. It's possible that a data chunk resets the timeout right before it expires without completing the request.

I'll open a separate PR if I remember :)

Reply:
@lpinca Perhaps you or I can add a squash-able commit here to change it to `Milliseconds of inactivity before a request times out.`?